target/linux/ramips/image/mt7621.mk

```diff
@@ -443,66 +443,74 @@ define Device/netgear_ex6150
 endef
 TARGET_DEVICES += netgear_ex6150
 
-define Device/netgear_r6220
+define Device/netgear_sercomm_nand
   MTK_SOC := mt7621
   BLOCKSIZE := 128k
   PAGESIZE := 2048
   KERNEL_SIZE := 4096k
-  IMAGE_SIZE := 28672k
   UBINIZE_OPTS := -E 5
-  SERCOMM_HWID := AYA
-  SERCOMM_HWVER := A001
-  SERCOMM_SWVER := 0x0086
+  SERCOMM_HWNAME :=
+  SERCOMM_HWID :=
+  SERCOMM_HWVER :=
+  SERCOMM_SWVER :=
```

Comment:
I'm not a magician in this field - but this should also work without these empty assignments (or am I missing something here)?

Reply:
I'm never sure with those definitions and DEVICE_VARS, so I copied from tplink in mt76x8:
https://github.com/openwrt/openwrt/blob/master/target/linux/ramips/image/mt76x8.mk#L250

I suspect you are right and the empty variables can be removed, but I'm not sure enough and cannot test it without the devices.

If _you_ are sure enough for both of us, I will be glad to remove the useless empty definitions.

Reply:
I've had a deeper look and it seems that I can indeed remove all four "empty" definitions.

Reply:
I also had a look at the introduction of the empty tplink variables in mt76x8.mk. To me it looks like the empty variables were put there just to indicate which variables need to be set for derived devices (at least I interpret the commit message that way).

https://github.com/openwrt/openwrt/commit/8ca650b8ce22fe2175cf15cfdd0e25206dd77481